hw1/libs/argv.js

```diff
@@ -0,0 +1,27 @@
+const yargs = require('yargs/yargs')
+const { hideBin } = require('yargs/helpers')
+const os = require('os');
+const ip = require('ip');
+
+const argv = yargs(hideBin(process.argv)).argv
+
+let res = {}
+if(argv.cpu){
+    res.cpu = os.arch();
```

Comment:
```suggestion
    res.cpu = os.cpus();
```
Но по заданию надо было вывести модель и количество ядер, так что ещё надо подправить)